src/components/InlineWrapper/InlineWrapper.scss

```diff
@@ -0,0 +1,22 @@
+@import '~@quartz/styles/scss/borders';
+@import '~@quartz/styles/scss/media-queries';
+@import '~@quartz/styles/scss/fonts';
+
+.container {
+	@include font-maison-500-3;
+
+	background-color: $color-background-2;
+	border: $border-solid-decorative;
+	border-radius: $size-border-radius;
+	margin: 20px 0 40px 0;
```

Comment:
```suggestion
```
Let's keep the outer spacing the responsibility of the consumer (qz-react now has some handy SCSS tokens for this)